lesson3/Lesson_3/app/src/main/res/layout/fragment_main.xml

```diff
@@ -23,46 +23,46 @@
             <TextView
                 android:layout_width="match_parent"
                 android:layout_height="75dp"
-                android:text="To Do List"
+                android:text="@string/to_do_list"
```

Comment:
Remember to extract all your string resources to strings.xml for all user-facing strings!  It's really easy, Android Studio will give you a pop-up saying that you should extract string resource and all you need to do is hit `Alt + Enter` and it'll do it for you.